coordinator/src/trade/mod.rs

```diff
@@ -678,39 +677,52 @@ impl TradeExecutor {
 
         let peer_id = trade_params.pubkey;
 
+        // Update position based on the outstanding funding fee events _before_ applying resize.
+        let funding_fee_events =
+            db::funding_fee_events::get_outstanding_fees(conn, position.trader, position.id)?;
+
+        let funding_fee = funding_fee_from_funding_fee_events(&funding_fee_events);
+
         let maintenance_margin_rate = {
             Decimal::try_from(self.node.settings.read().await.maintenance_margin_rate)
                 .expect("to fit")
         };
 
+        let position = position.apply_funding_fee(funding_fee, maintenance_margin_rate);
+
+        let (original_collateral_reserve_coordinator, original_collateral_reserve_trader) = self
```

Comment:
I wanted to highlight that this value corresponds to the collateral reserve _before_ resizing. This is useful in the log message below, but I can add the extra context there so that the variable name is not confusing.